examples/hippy-react-demo/scripts/hippy-webpack.dev.js

```diff
@@ -2,20 +2,34 @@ const fs = require('fs');
 const path = require('path');
 const webpack = require('webpack');
 const HippyDynamicImportPlugin = require('@hippy/hippy-dynamic-import-plugin');
+const HippyHMRPlugin = require('@hippy/hippy-hmr-plugin');
+const ReactRefreshWebpackPlugin = require('@hippy/hippy-react-refresh-webpack-plugin');
 const pkg = require('../package.json');
+
 module.exports = {
   mode: 'development',
   devtool: 'eval-source-map',
   watch: true,
   watchOptions: {
     aggregateTimeout: 1500,
   },
+  devServer: {
+    port: 38988,
+    hot: true,
+    devMiddleware: {
+      writeToDisk: true,
+    },
+    client: {
+      overlay: false,
+    },
+  },
   entry: {
     index: ['regenerator-runtime', path.resolve(pkg.main), '@hippy/hippy-live-reload-polyfill'],
   },
   output: {
     filename: 'index.bundle',
     strictModuleExceptionHandling: true,
+    publicPath: 'http://localhost:38989/',
```

Comment:
why use publicPath

Reply:
had remove this config, add `hotManifestPublicPath` option to `HippyHMRPlugin`